src/inmanta/config.py

```diff
@@ -92,6 +93,7 @@ def load_config(
         config = LenientConfigParser(interpolation=Interpolation())
         config.read(files)
         cls.__instance = config
+        cls._config_dir = config_dir
```

Comment:
We should also reset it in the `_reset` method

Reply:
thx!

Reply:
done